third_party/flower/src/proto/flwr/proto/BUILD.gn

```diff
@@ -0,0 +1,12 @@
+import("//third_party/grpc/grpc_library.gni")
```

Comment:
Could we avoid using grpc at all?

It don't see any grpc call in the added code, so I suppose we do it only for comping `flower`. Could we simple cut all rpc methods?

Reply:
that's true. I removed all rpc methods. But I'm having some difficulties compiling the library via `proto_library` instead of `grpc_library`. Seeing some undefined symbols issues when compiling (`undefined symbol: google::protobuf::Message:...`). Might be worth having a look at this together